util/database.ts

```diff
@@ -0,0 +1,14 @@
+import { DataSource } from "typeorm"
+import {Movie} from "../model/Movie"
+
+export const AppSource = new DataSource({
+    type: "postgres",
+    host: "localhost",
+    port: 5432,
+    username: "postgres",
+    password: "martina",
+    database: "moviesDb",
+    entities:[Movie],
+    synchronize: true
```

Comment:
Using synchronize: true is okay for development, but should not be used for production. Try and set it to false and use/write database migrations.